gtsfm/data_association/data_assoc.py

```diff
@@ -39,12 +36,16 @@ class DataAssociation(NamedTuple):
         min_track_len: min length required for valid feature track / min nb of supporting views required for a landmark
                        to be valid.
         mode: triangulation mode, which dictates whether or not to use robust estimation.
+        min_angle_thresh (optional): threshold of the angle at the 3D point by backprojected rays from 2 cameras for
```

Comment:
thanks for the updates, Ayush. I prefer something along the lines of `min_tri_angle_deg` like COLMAP.

How about: 
```python
min_tri_angle_deg (optional): minimum allowed angle in degrees between any two rays
     during triangulation.  Each ray originates at a camera and shoots towards the triangulated 3d point.
     Small angles indicate small baselines, which are unreliable. Defaults to None (check inactive).
```

Reply:
Theia also uses something along these lines: `min_triangulation_angle_degrees`